source/lib/courses/load-all-courses.js

```diff
@@ -6,9 +6,14 @@
 
 import type {CourseCollectionType} from './types'
 import {loadStudentNumber} from './load-student-number'
-import {loadCoursesFromUnofficialTranscript} from './load-courses-from-unofficial-transcript'
+import {
+  loadCoursesFromUnofficialTranscript,
+} from './load-courses-from-unofficial-transcript'
 
-export async function loadAllCourses(isConnected: boolean, force?: boolean): Promise<CourseCollectionType> {
+export async function loadAllCourses(
+  isConnected: boolean,
+  force?: boolean,
```

Comment:
You might ask, "Are trailing commas valid in function parameter lists?"

The answer is, "kinda". They've been accepted for ES2017 (https://github.com/tc39/proposal-trailing-function-commas), but you can't really use them yet. However, we use React Native, which compiles all our JS code anyway, and they include a compiler for the trailing function commas, so we get to use them!

All that is to say, yes, they are valid for use to use. 😄